.travis.yml

```diff
@@ -1,14 +1,18 @@
 language: python
 python:
   - "2.7"
-  - "3.3"
   - "3.4"
   - "3.5"
   - "3.6"
```

Comment:
Don't we need 3.7 here?

Reply:
3.7 was not available out of the box. So was installing it explicitly in the next few lines. But, looks like support got added a few hours ago(https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-438518560). Will fix it now.